app/models/early_career_teacher_profile.rb

```diff
@@ -15,7 +15,7 @@ class EarlyCareerTeacherProfile < ApplicationRecord
   scope :pupil_premium, -> { where(pupil_premium_uplift: true) }
   scope :uplift, -> { sparsity.or(pupil_premium) }
 
-  after_save :save_participant_profile
+  # after_save :save_participant_profile
```

Comment:
Is this commented out on purpose? Ditto for the mentor profile

Reply:
Oof, definitely not - commented it out just to test the task...